src/controls/QskComboBox.cpp

```diff
@@ -0,0 +1,276 @@
+/******************************************************************************
+ * QSkinny - Copyright (C) 2023 Uwe Rathmann
+ * This file may be used under the terms of the QSkinny License, Version 1.0
+ *****************************************************************************/
+
+#include "QskComboBox.h"
+
+#include "QskGraphic.h"
+#include "QskMenu.h"
+#include "QskTextOptions.h"
+
+#include <QGuiApplication>
+
+QSK_SUBCONTROL( QskComboBox, Panel )
+QSK_SUBCONTROL( QskComboBox, Graphic )
+QSK_SUBCONTROL( QskComboBox, Text )
+QSK_SUBCONTROL( QskComboBox, OpenMenuGraphic )
+QSK_SUBCONTROL( QskComboBox, Ripple )
+
+QSK_SYSTEM_STATE( QskComboBox, Pressed, QskAspect::FirstSystemState << 1 )
+QSK_SYSTEM_STATE( QskComboBox, MenuOpen, QskAspect::FirstSystemState << 2 )
+
+class QskComboBox::PrivateData
+{
+  public:
+    PrivateData( QskComboBox* const box )
+        : menu( new QskMenu( box ) )
+    {
+        menu->setPopupFlag( QskPopup::DeleteOnClose, false );
```

Comment:
menu becomes a dangling pointer after the popup has been closed ( DeleteOnClose ). Guess there will be crashes, when using it a second time ?

Reply:
No, we are setting the flag to false, so it will still be around after closing, right?